frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -151,6 +152,15 @@ export default function SideNavigation({
               icon={<ContactSupportIcon />}
             />
             <Divider />
+            <NavMenuItem
+              href='/docs/python/index.html'
+              selectedPage={page}
+              primaryText='Python Client Docs'
+              drawerOpen={drawerOpen}
+              menuPage=''
```

Comment:
This is required so that we can compare it to `selectedPage`. If they match then we highlight this tab as the selected tab. Some unique text like `menuPage='pythonDocs'` should be good.